### PR TITLE
[KOGITO-5290] Kogito-pmml quickstart

### PR DESCRIPTION
**Pull request by gitgabrio**
Kogito-pmml quickstart project

### CODE REVIEW

**Review by gsmet — APPROVED** [review]
I squashed the 2 commits. Let's wait for CI and merge.

File: kogito-pmml-quickstart/src/test/java/org/kie/pmml/kogito/quarkus/LogisticRegressionIrisDataTest.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    @Test
+    void testEvaluateLogisticRegressionIrisDataResult() {
+        given()
+                .contentType(ContentType.JSON)
+                .body(INPUT_DATA)
+                .when()
+                .post(BASE_PATH)
+                .then()
+                .statusCode(200)
+                .body(TARGET, is(52.5f));
```
Comment (gsmet) [inline kogito-pmml-quickstart/src/test/java/org/kie/pmml/kogito/quarkus/LogisticRegressionIrisDataTest.java:52]: AFAICS, this is incorrect and should be `"virginica"`. I fixed it and I'm running the native tests to see how it goes. I also upgraded the PR to 1.9.0.

**Review by gsmet — CHANGES_REQUESTED** [review]

File: kogito-pmml-quickstart/src/test/java/org/kie/pmml/kogito/quarkus/LogisticRegressionIrisDataTest.java (hunk 1 above)
Comment (gitgabrio) [reply kogito-pmml-quickstart/src/test/java/org/kie/pmml/kogito/quarkus/LogisticRegressionIrisDataTest.java]: Thanks @gsmet 
There were  also an issue about start imports - I hope it is fixed now

**Comment by gsmet** [comment]
I had fixed everything in my commit. Let me fix the history as you made it pretty nasty.

**Comment by gsmet** [comment]
Force pushed to fix the history. Let's wait for CI.

**Comment by gitgabrio** [comment]
> I had fixed everything in my commit. Let me fix the history as you made it pretty nasty.

Yes, usually when I receive a request for change I made changes and push them...

**Comment by gsmet** [comment]
I wasn't sure if you were around given August is there :). FYI, I'll have a look to the others in the afternoon.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Merged, thanks!

**Comment by gitgabrio** [comment]
Thanks! 
Best
Gabriele

**Pull request merged**